### ISSUE TITLE
Adding custom styles

### ISSUE THREAD

**Issue by leem32** [umputun/remark#292]
I'd like to add some custom styles to make remark fit better with my page.
According to [this remark issue](https://github.com/umputun/remark/issues/267#issuecomment-457924716), it's possible to add a custom stylesheet like this:

```javascript
  var remark_config = {
    site_id: 'YOUR_SITE_ID',
    ...
    style: "https://somehost.com/remark.css"
  };
```

But it doesn't seem to be working for me. Ive tried both relative and absolute URL's e.g `https://somehost.com/css/remark.css` and `/css/remark.cs`, `./css/remark.css`.

Am I doing it wrong?

**Comment by umputun** [comment]
#267 you mentioned is work in progress. I'm not sure if passing style supported in the current version. @Reeywhaar - am I right, or right ;)

**Comment by Reeywhaar** [comment]
@umputun no, you're right! I made bare minimum in refactoring, to avoid long review (still there is), I am planning to make other features after refactoring will be accepted (#279, external styles, #285, etc...)

**Comment by umputun** [comment]
closed as dup #5

**Comment by leem32** [comment]
I thought that might be the case. 

In the meantime, I'll just add the styles using pure JS instead.

Here's a couple of examples on how to do that. They work by first detecting when the `iframe` is loaded in the dom and then uses an `eventListener` to detect when the `iframe` content has loaded and injects the stylesheet into the `head` of the `iframe` html.. 

From my tests both seem to work equally well so just choose the one you prefer.

Example 1

```javascript

var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
var observer = new MutationObserver(function(mutations) {

    mutations.forEach(function (mutation) {

        [].filter.call(mutation.addedNodes, function (node) {

            return node.nodeName == 'IFRAME';

        }).forEach(function (node) {

            node.addEventListener('load', function (e) {

                console.log("iframe content loaded");

                var iframe = document.querySelector('#remark42 iframe');

                // style iframe element
                // iframe.style.border = "5px solid aqua";

                var style = document.createElement('style');
                style.id = "custom-remark-styles";

                style.textContent =
                  'body {' +
                  '  background-color: green;' +
                  '  font-weight: italic;' +
                  '}'
                ;

                iframe.contentDocument.head.appendChild(style);
                console.log(iframe.contentDocument.head);

                // and disconnect
                observer.disconnect();

            });
        });
    });
});

var targetNode = document.getElementById("remark42");
observer.observe(targetNode, { childList: true, subtree: true });

// also works
// observer.observe(document.body, { childList: true, subtree: true });

```

Example 2
This example also uses the MutationObserver constructor but is more readable.

```javascript

var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
var observer = new MutationObserver(function(mutations) {

    if (mutations[0].type === "childList") { // optional

        if (document.querySelector("#remark42 iframe")) {
            
            console.log("iframe added to dom");

            var iframe = document.querySelector('#remark42 iframe');

            iframe.addEventListener('load', function (e) {

                console.log("iframe content loaded");

                // style iframe element
                // iframe.style.border = "5px solid aqua";

                var style = document.createElement('style');
                style.id = "custom-remark-styles";

                style.textContent =
                  'body {' +
                  '  background-color: green;' +
                  '  font-weight: italic;' +
                  '}'
                ;

                iframe.contentDocument.head.appendChild(style);
                // console.log(iframe.contentDocument.head);

                // and disconnect
                observer.disconnect();

            });
        }
      }
});

var observerConfig = {
    childList: true,
};

var targetNode = document.getElementById("remark42");
observer.observe(targetNode, observerConfig);

```
